addons/mail/static/src/core/common/thread.js

```diff
@@ -440,7 +441,8 @@ export class Thread extends Component {
 
     updateShowJumpPresent() {
         this.state.showJumpPresent =
-            this.props.thread.loadNewer || this.presentThresholdState.isVisible === false;
+            this.props.thread.loadNewer ||
+            (this.presentThresholdState.isVisible === false && this.presentThresholdRef.el);
```

Comment:
I'm wondering if this shouldn't be part of `useVisible` instead.

`isVisible` should become `undefined` rather than `false` when the ref is gone, similar to what is done for the initial case.

Reply:
Indeed better, I changed the code :)